src/apps/answers/errors.py

```diff
@@ -90,3 +90,8 @@ class MultiinformantAssessmentNoAccessApplet(ValidationError):
 class MultiinformantAssessmentInvalidActivityOrFlow(ValidationError):
     message = _("Activity or Flow not found")
     code = _("invalid_activity_or_flow_id")
+
+
+class MultiinformantAssessmentMissingTargetAndSource(ValidationError):
+    message = _("Both target_subject_id and source_subject_id must be provided")
+    code = _("missing_target_and_source")
```

Comment:
This class is no longer needed, since the root validator introduced below is overkill (we should just use built-in Pydantic validation).

Reply:
Thanks Paul, I deleted the class